### PR TITLE
Trip details are saved and displayed in the TripIndicatorActivity.

### PR DESCRIPTION
**Pull request by free4murad**
Solves issue #193 
@yatna @chhavip 
Please review the PR

### CODE REVIEW

File: malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java [hunk 1]
```diff
@@ -311,6 +330,19 @@ else if(tripTime.getText().toString().equals(""))
...
                    if(deptime>today) {
+                       //Saving the trip details.
+                       SharedPreferences.Editor editTripDetails = getSharedPreferences("trip-details", MODE_MULTI_PROCESS).edit();
+                       editTripDetails.putString("destination", mLocationPicked);
+                       editTripDetails.putString("departure_month", ((TextView) findViewById(R.id.trip_month_departure)).getText().toString());
+                       editTripDetails.putString("departure_date", ((TextView) findViewById(R.id.trip_date_departure)).getText().toString());
+                       editTripDetails.putString("departure_year", ((TextView) findViewById(R.id.trip_year_departure)).getText().toString());
+                       editTripDetails.putString("arrival_month", ((TextView) findViewById(R.id.trip_month)).getText().toString());
+                       editTripDetails.putString("arrival_date", ((TextView) findViewById(R.id.trip_date)).getText().toString());
+                       editTripDetails.putString("arrival_year", ((TextView) findViewById(R.id.trip_year)).getText().toString());
```
Comment (chhavip) [inline malaria-app-android/src/main/java/com/peacecorps/malaria/activities/TripIndicatorFragmentActivity.java]: Define all shared preferences keys in strings.xml and use from there.

**Comment by free4murad** [comment]
@chhavip 
done, please review.

**Comment by chhavip** [comment]
@free4murad I just noticed this PR has been sent against the master branch of the project, please send it against the develop branch to be merged.

**Comment by free4murad** [comment]
@chhavip  
Done

**Comment by free4murad** [comment]
@chhavip 
i see there are some extra commits.
Actually I am facing problems with rebasing my repo.
Can you please merge #165 so that I can delete my repo and fork it again?

**Comment by free4murad** [comment]
@yatna @chhavip
I have changed the base branch to `develop`.
Also I have removed multiple commits.
Please review.

**Pull request merged**